src/IntegralArrays.jl

```diff
@@ -1,5 +1,78 @@
 module IntegralArrays
 
 # package code goes here
+using Images, ClosedIntervals
```

Comment:
I'm not sure where to put it. Maybe https://github.com/JuliaMath.

And yes, I'd use the code for UnitRanges as the main inspiration: I'm thinking of these as ranges without any step. Obviously, don't mimic this:
```julia
julia> 0:-5
0:-1
```
just keep whatever the user specified.